rfcs/0175-appeals-council.md

```diff
@@ -0,0 +1,165 @@
+- Establish an *Appeals Council* to hold moderation team members and procedures accountable and to allow for the revision of significant decisions if they fail to meet the community's reasonable standards.
+
+## Memorandum of Understanding on Equitable Moderation
+
+In response to the impression left by past events and with a fervent desire for improvement, we adopt the following principles and maxims as the *Memorandum of Understanding on Equitable Moderation*. These principles serve to clarify the overlapping consensus on reasonable expectations for community participation:
+
+1) We, as moderators, commit to basing decisions on general principles and concrete reasons that apply equally to all community members, irrespective of characteristics such as sociopolitical orientation, economic status, identity, or religion.
+2) We adhere to the principles of *equality* and *objectivity*. Equality refers to the status of each community member, while objectivity pertains to the consistent, unbiased reasoning behind our decisions.
+3) A *concrete reason* is a verifiable state of affairs that can be demonstrated to other community members upon request.
+4) We acknowledge that *concrete reasons* are not self-explanatory, except in extreme cases, and we will provide explanations relating them to the general principles guiding our decisions.
+5) We affirm that moderation actions must be fair and proportionate to the related issue.
+6) We ensure that concrete reasons for permanent bans are available for inspection by long-standing community members.
+7) We recognize that moderation actions should not be based solely on the volume of complaints.
+8) We commit to treating community members with respect and avoiding derogatory language when addressing concerns.
+9) We affirm our role as moderators is to facilitate a respectful community, not to punish members. We will use language that reflects mutual respect.
+10) We prefer inclusion to exclusion, resorting to permanent exclusion only in cases of extreme necessity. We are committed to exploring ways to prevent exclusion and to engaging constructively in efforts to reinstate excluded members.
+11) To reflect a consensus in a diverse community, the moderation team must embody cultural and viewpoint diversity, respecting members with varying societal views, provided they do not undermine equal community membership.
+12) We respect viewpoint diversity in cultural and political matters when selecting moderation team candidates.
+13) We understand that reasonable moderation is based on past events or future expectations, necessitating respect for historical records and transparency in controversial events.
+14) We uphold that true diversity necessitates exceptional tolerance, actively embracing not only the ideas we favor but, also, those we disagree with. This commitment to tolerance is fundamental in fostering a community where diverse viewpoints can coexist and enrich our collective experience.
```

Comment:
I'm afraid you might have a flawed conception of what the *desired core* of diversity means.

The diversity a community needs isn't about either trying to widen the spectrum as wide as possible or gathering as many arbitrarily defined sub groups just for the sake of some metric, but it's about fostering an environment where also those viewpoints that would otherwise not dare to be there *because of the presence of certain others* can come up.
In this sense, it can even paradoxically increase diversity when 2 usually widely-represented factions are restricted just to enable only 1 other faction to be present in the community – if it is one faction that requires an exceptional safe environment to even be heard.

Reply:
> I would've pointed you towards the paradox of intolerance

And that ~distorted~ "simplification" from [PictOnline](https://twitter.com/pictoline/status/897260210644852737/photo/1)? Pfft.